src/operator/subgraph/tensorrt/tensorrt-inl.h

```diff
@@ -267,6 +267,24 @@ class TensorrtProperty : public SubgraphProperty {
     return std::make_shared<TensorrtProperty>();
   }
 
+  void PrePartition(const nnvm::Graph& g,
+    const std::vector<std::pair<std::string, std::string>>& options_map) override {
+    auto& in_arg_names = g.GetAttr<std::vector<std::string>>("in_arg_names");
+    auto& in_aux_names = g.GetAttr<std::vector<std::string>>("in_aux_names");
+    NDArray **in_args_ptr = g.GetAttr<NDArray**>("in_args");
+    NDArray **in_aux_ptr = g.GetAttr<NDArray**>("in_aux");
+    // should we check if not empty?
```

Comment:
we dont need to check if they are not empty. we clear them in case the user calls partitioning twice in the same process. MXNet only instantiates one subgraph property instance for each class and keeps it in a map and reuses it. So we have to clean up from the previous partitioning call.

Reply:
Removed the comment